frontend/src/metabase/nav/components/WhatsNewNotification/utils.unit.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { createMockVersionInfoRecord } from "metabase-types/api/mocks";
+import { getLatestEligibleReleaseNotes } from "./utils";
+
+describe("What's new - utils", () => {
+  describe("getLatestEligibleReleaseNotes", () => {
+    it("doesn't filter old versions if lastAck is null", () => {
+      expect(
+        getLatestEligibleReleaseNotes({
+          versions: [
+            createMockVersionInfoRecord({ version: "v0.48" }),
+            createMockVersionInfoRecord({
+              version: "v0.43",
+              announcement_url: "url",
+            }),
+          ],
+          currentVersion: "v0.48",
+          lastAcknowledgedVersion: null,
+        }),
+      ).toHaveProperty("version", "v0.43");
+    });
+
+    it("filters out ack versions", () => {
+      expect(
+        getLatestEligibleReleaseNotes({
+          versions: [
+            createMockVersionInfoRecord({
+              version: "v0.47",
+              announcement_url: "url",
+            }),
+          ],
+          currentVersion: "v0.48",
+          lastAcknowledgedVersion: "v0.47",
+        }),
+      ).toBe(undefined);
+
+      expect(
+        getLatestEligibleReleaseNotes({
+          versions: [
+            createMockVersionInfoRecord({
+              version: "v0.46",
+              announcement_url: "url",
+            }),
+          ],
+          currentVersion: "v0.48",
+          lastAcknowledgedVersion: "v0.47",
+        }),
+      ).toBe(undefined);
+    });
+
+    it("returns up to the current version", () => {
+      expect(
+        getLatestEligibleReleaseNotes({
+          versions: [
+            createMockVersionInfoRecord({
+              version: "v0.48",
+              announcement_url: "url",
+            }),
+          ],
+          currentVersion: "v0.48",
+          lastAcknowledgedVersion: "v0.47",
+        }),
+      ).not.toBe(undefined);
+    });
+
+    it("filters out future versions", () => {
```

Comment:
This test doesn't work as expected. The version record is filtered out because there is no version record for v0.47